aspnetcore/web-api/advanced/custom-formatters.md

```diff
@@ -10,13 +10,17 @@ uid: web-api/advanced/custom-formatters
 
 By [Tom Dykstra](https://github.com/tdykstra)
```

Comment:
Feel free to add your name here.

Reply:
Thanks for the suggestion. I don't really feel comfortable with being named on something I've not had much involvement in, as awesome as it is to see my name on these things. :trophy: